core/global.d.ts

```diff
@@ -11,7 +11,12 @@ type LuaUserdata = { readonly __internal__: unique symbol };
  * A global variable (not a function) that holds a string containing the running
  * Lua version.
  */
-declare const _VERSION: 'Lua 5.1' | 'Lua 5.2' | 'Lua 5.3' | 'Lua 5.4';
+declare const _VERSION:
+    | ('Lua 5.0' | 'Lua 5.0.1' | 'Lua 5.0.2' | 'Lua 5.0.3')
```

Comment:
We also didn't add any subversions for the other versions, so should we do it for 5.0?

Reply:
Good point. `_VERSION` maps to the C Macro `LUA_VERSION`. AFAIK this has always been defined as `Lua Major.Minor`. So you should probably get rid of the additional subversions/patch releases.

Reply:
Nope, in Lua 5.0, the subversions are part of the version string. So it is necessary to test against all of them.

Here is the value of _VERSION as defined in [5.0.3](https://www.lua.org/ftp/lua-5.0.3.tar.gz):

![image](https://user-images.githubusercontent.com/4701008/183941708-1aa3611a-137a-41da-9be1-2a0c8ddfdf75.png)

Compare to the [current way}(https://github.com/lua/lua/blob/d61b0c60287c38008d312ddd11724a15b1737f7b/lua.h#L26) it is defined, which uses major.minor.